config.yaml

```diff
@@ -1,3 +1,7 @@
-foo:
-  string: foo
-  int: 12345
+fileserver:
+  address: localhost
+  port: 12345
```

Comment:
I don't think we want the fileserver to be separate from the main server hosting the API, I don't think we want a separate section for the fileserver here.